app/views/editorial/news/index.html.haml

```diff
@@ -0,0 +1,8 @@
+%h1 News
+
+%p
+  = link_to('Create news article', editorial_new_news_article_path)
+
+TODO: filters
```

Comment:
I'm fine with people seeing it rendered so they're aware there's more to come, but I'm also happy to make it a comment if people are worried about introducing noise into our production app.